frontend/src/components/dashboard/MineList.js

```diff
@@ -77,7 +99,7 @@ const columns = [
     dataIndex: "commodity",
     width: 150,
     render: (text, record) => (
-      <div>
+      <div className="td-inner" data-label="Commodity" title="Commodity">
```

Comment:
Is `title` doing anything here? (And on all the divs where its been added?)

Reply:
We have some ambiguous naming conventions here (see TSF) which I've added a title tag for accessibility purposes. I've translated this across to each "td-inner" div for consistency.